client/App.tsx

```diff
@@ -42,9 +67,78 @@ export default function HomePage() {
                 </button>
             </div>
             <div className="container">
-                <section id="pageOne" />
+                <section id="pageOne">
+                    <div id="pageOneTextBox">
+                        <h1>Graph QL Rate Limiter</h1>
+                        <h2>Gateway visualisation tool</h2>
+                        <p>
+                            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+                            tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
+                            veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea
+                            commodo consequat.
+                        </p>
+                    </div>
+                </section>
+                <section id="demoCard">
```

Comment:
What is the plan for a responsive Web Design? The `demoCard` element overlaps the "What makes us different" section on a shorter screen. 

I also see a gap about the height of my screen between the demo card and "What Makes different" sections when I am at full window.

Reply:
Not worked on a responsive part yet. The gap is there on the purpose ( demo card is "above" the rest of the page)